extensions/slack/src/shared/client/SlackClient.ts

```diff
@@ -56,10 +62,31 @@ export class SlackClient {
         .filter(
           ({ is_bot, is_workflow_bot, deleted, id }) => !is_bot && !is_workflow_bot && !deleted && id !== "USLACKBOT",
         )
-        .map(({ id, name: username, profile, team_id }) => {
+        .map(({ id, name: username, profile, team_id, tz }) => {
           const firstName = profile?.first_name ?? "";
           const lastName = profile?.last_name ?? "";
           const name = `${firstName} ${lastName}`;
+          const jobTitle = profile?.title ?? "";
+          const statusEmoji = profile?.status_emoji ?? undefined;
+          const statusText = profile?.status_text?.replace(/&amp;/g, "&") ?? undefined;
+          const statusExpiration = profile?.status_expiration ?? undefined;
+          const timezone = tz ?? "";
+
+          let statusExpirationDate = "";
+          let statusExpirationTime = "";
+
+          if (statusExpiration) {
+            const date = new Date(statusExpiration * 1000);
+            if (!isNaN(date.getTime())) {
+              statusExpirationDate = date.toISOString().split("T")[0];
+              statusExpirationTime = date.toLocaleTimeString(getTimeLocale(), {
+                hour: "2-digit",
+                minute: "2-digit",
+              });
+
+              statusExpirationDate = `Back on ${statusExpirationDate} at ${statusExpirationTime}`;
```

Comment:
Would it be possible to format the date in a way that's more human-readable (such as [formatRelative](https://date-fns.org/v3.6.0/docs/formatRelative) from `date-fns`)

Reply:
This was a great suggestion. I think this is what Slack does as well. 
![date-fns - modern JavaScript date utility library-2024-07-18-21 52 37@2x](https://github.com/user-attachments/assets/6cda90b0-b8b0-4fb0-b5c3-1f0ef5a41fc8)